City-of-Helsinki/drupal-hdbt
# UHF-405: Add supports Swedish -styling and improve styling based on d…

…esign feedback

# Style in units the "Service in Swedish" -text

[UHF-405](https://helsinkisolutionoffice.atlassian.net/browse/UHF-405)

## What was done

* Discussed with design about the original specifications and found a better alternative
* Styled the "Service in Swedish" tag in these contexts:
  * unit
  * teaser
  * teaser with image 
  * (but not to teaser wide, because Elina asked not to)
* Fixed teaser and teaser with image margins, paddings font-sizes etc to match current design specs (linked in [jira issue comment](https://helsinkisolutionoffice.atlassian.net/browse/UHF-405?focusedCommentId=47237))

Unit page desktop:
![image](https://user-images.githubusercontent.com/1191667/135230614-56b88fa9-c294-4451-95a5-d3c6a1e6666d.png)
Unit page mobile:
![image](https://user-images.githubusercontent.com/1191667/135230668-030bfecc-329c-4401-8e19-c6b1863e1455.png)
Teaser desktop:
![image](https://user-images.githubusercontent.com/1191667/135230714-4119ef1a-7175-4ef9-ae6f-67f0b08dd334.png)
Teaser mobile:
![image](https://user-images.githubusercontent.com/1191667/135230801-61e5a5c7-5a41-47ae-b6e9-1cda6170609b.png)
Teaser with image desktop:
![image](https://user-images.githubusercontent.com/1191667/135230879-23c28a65-beb5-43df-8047-47754fb665a2.png)
Teaser with image mobile:
![image](https://user-images.githubusercontent.com/1191667/135230954-a6e362a6-90dc-4304-86d6-04fd6552236d.png)


## How to test

- Run SOTE instance
- Get the helfi_tpr module branch [as instructed in its PR](https://github.com/City-of-Helsinki/drupal-module-helfi-tpr/pull/55)
- Get this branch `composer require drupal/hdbt:dev-UHF-405_unit_service_in_lang_sv`
- Check that these pages both in desktop and mobile sizes have the yellow tag [as instructed in specs](https://helsinkisolutionoffice.atlassian.net/browse/UHF-405?focusedCommentId=47237):
  - Unit page
  - Unit teaser with image
  - Unit teaser
- Check that the same content behaves well without the yellow tag (for example teaser with image has min-height)

Related: [helfi_tpr module PR](https://github.com/City-of-Helsinki/drupal-module-helfi-tpr/pull/55)
